integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDynamicUpdate.java

```diff
@@ -549,33 +550,158 @@ public void testMiiChangeDomainName() {
     // Patch a running domain with introspectVersion.
     patchDomainResourceWithNewIntrospectVersion(domainUid, domainNamespace);
 
+    // Verifying introspector pod is created and failed
+    logger.info("verifying the introspector failed with the expected error msg");
+    verifyIntrospectorFailsWithExpectedErrorMsg(MII_DYNAMIC_UPDATE_EXPECTED_ERROR_MSG);
+
+    // clean failed introspector
+    // replace WDT config map with model.config.wm.yaml because we can not delete the entire WM tree
+    // there is an issue that the application SourcePath can not be found when the target is removed
+    // https://jira.oraclecorp.com/jira/browse/WDT-535
+    Path pathToAppDeploymentYaml = Paths.get(WORK_DIR + "/appdeployment.yaml");
+    String yamlToAppDeployment = "appDeployments:\n"
+        + "  Application:\n"
+        + "    myear:\n"
+        + "      Target: 'cluster-1'";
+
+    assertDoesNotThrow(() -> Files.write(pathToAppDeploymentYaml, yamlToAppDeployment.getBytes()));
+
+    replaceConfigMapWithModelFiles(configMapName, domainUid, domainNamespace,
+        Arrays.asList(MODEL_DIR + "/model.config.wm.yaml", pathToAppDeploymentYaml.toString()),
+        withStandardRetryPolicy);
+
+    // Patch a running domain with introspectVersion.
+    patchDomainResourceWithNewIntrospectVersion(domainUid, domainNamespace);
+
+    // Verifying introspector pod is created, runs and deleted
+    verifyIntrospectorRuns();
+  }
+
+  /**
+   * Negative test: Changing the listen port of a server using mii dynamic update.
```

Comment:
Can you describe details of the expected outcome in this case.

Reply:
fix. 
To answer your question above, all the error msgs are the same.